packages/ng/pagination/pagination.component.html

```diff
@@ -0,0 +1,19 @@
+<nav class="pagination" role="navigation" aria-label="pagination-count" [class.mod-compact]="mod() === 'compact'">
+	@if (mod() === 'default') {
+	<div class="pagination-count">
+		<span class="u-mask"> {{ intl.resultsA11y | intlParams: { from: from(), to: to(), itemsCount: itemsCount() } }} </span>
+		<span aria-hidden="true"> {{ intl.results | intlParams: {from: from(), to: to(), itemsCount: itemsCount()} }} </span>
+	</div>
+	}
+
+	<div class="pagination-scrolling">
+		<button type="button" class="button mod-onlyIcon mod-text mod-S" [disabled]="isFirstPage()" (click)="previousPage.emit()">
+			<span aria-hidden="true" class="lucca-icon icon-arrowChevronLeft"></span>
```

Comment:
Prefer `lu-icon` 
```suggestion
			<lu-icon icon="arrowChevronLeft"/>
```

Reply:
I just realized but you can also pass the alt text to the icon and make it a one-liner